From a4fd3a6b4be493db3b9e779e70f98d35d19a1e7b Mon Sep 17 00:00:00 2001
From: Robin Gong <yibin.gong@nxp.com>
Date: Tue, 16 May 2017 10:30:18 +0800
Subject: [PATCH 1788/5242] MLK-14925: ARM: dts: imx6qdl-sabresd-hdcp: fix
 gpu/vpu probe fail on hdcp board

commit  3bbe65a00be92d8a6ee4d0edeff16504547c4eeb from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix the same issue as commit bf23e0c5ea31 on imx6qp-saresd-hdcp board

Signed-off-by: Robin Gong <yibin.gong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6dl-sabresd-hdcp.dts |   17 +++++++++++++++++
 arch/arm/boot/dts/imx6q-sabresd-hdcp.dts  |   17 +++++++++++++++++
 2 files changed, 34 insertions(+)

diff --git a/arch/arm/boot/dts/imx6dl-sabresd-hdcp.dts b/arch/arm/boot/dts/imx6dl-sabresd-hdcp.dts
index 68bebbc..2c7f044 100644
--- a/arch/arm/boot/dts/imx6dl-sabresd-hdcp.dts
+++ b/arch/arm/boot/dts/imx6dl-sabresd-hdcp.dts
@@ -8,6 +8,11 @@
 
 #include "imx6dl-sabresd.dts"
 
+&gpc {
+	/* use ldo-enable, u-boot will check it and configure */
+	fsl,ldo-bypass = <0>;
+};
+
 &hdmi_video {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_hdmi_hdcp>;
@@ -17,3 +22,15 @@
 &i2c2 {
 	status = "disable";
 };
+
+&reg_arm {
+	/delete-property/ vin-supply;
+};
+
+&reg_pu {
+	/delete-property/ vin-supply;
+};
+
+&reg_soc {
+	/delete-property/ vin-supply;
+};
diff --git a/arch/arm/boot/dts/imx6q-sabresd-hdcp.dts b/arch/arm/boot/dts/imx6q-sabresd-hdcp.dts
index ba1fa33..3116e3e 100644
--- a/arch/arm/boot/dts/imx6q-sabresd-hdcp.dts
+++ b/arch/arm/boot/dts/imx6q-sabresd-hdcp.dts
@@ -12,6 +12,11 @@
 
 #include "imx6q-sabresd.dts"
 
+&gpc {
+	/* use ldo-enable, u-boot will check it and configure */
+	fsl,ldo-bypass = <0>;
+};
+
 &hdmi_video {
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_hdmi_hdcp>;
@@ -21,3 +26,15 @@
 &i2c2 {
 	status = "disable";
 };
+
+&reg_arm {
+	/delete-property/ vin-supply;
+};
+
+&reg_pu {
+	/delete-property/ vin-supply;
+};
+
+&reg_soc {
+	/delete-property/ vin-supply;
+};
-- 
1.7.9.5

